Gemfile

```diff
@@ -39,6 +39,7 @@ gem 'omniauth-rails_csrf_protection', '~> 0.1'
 gem 'haml-rails'
 gem 'addressable'
 gem 'normalize-rails'
+gem "autoprefixer-rails"
```

Comment:
According to https://github.com/ai/autoprefixer-rails#ruby-on-rails we need to run `rake tmp:clear` after introducing this - is that right?  If so would be helpful to mention this kind of thing in the commit message and on Slack to avoid people getting caught out when they `git pull`.

Reply:
Thanks. I missed that. I'll ping Slack.